packages/yourloops/pages/notifications/notification.tsx

```diff
@@ -294,32 +291,26 @@ export const Notification = (props: NotificationProps): JSX.Element => {
             }}
           />}
         {isAMonitoringInvitation && displayMonitoringTerms && notification.target &&
-          <MonitoringConsentDialog onAccept={acceptTerms} onCancel={() => { setDisplayMonitoringTerms(false) }}
-                                   teamName={notification.target.name} />
+          <MonitoringConsentDialog
+            onAccept={acceptTerms}
+            teamName={notification.target.name}
+            onCancel={() => {
+              setDisplayMonitoringTerms(false)
+            }}
+          />
         }
-        {props.userRole === UserRoles.caregiver && notification.type === NotificationType.careTeamProInvitation ? (
-          <IconButton
-            id={`notification-help-${id}-button`}
-            className="notification-help-button"
-            size="medium"
-            color="primary"
-            aria-label="notification-help-button"
-            onClick={props.onHelp}>
-            <HelpIcon id={`notification-help-${id}-icon`} />
-          </IconButton>
-        ) : (
-          <Button
-            id={`notification-button-accept-${id}`}
-            color="primary"
-            variant="contained"
-            disableElevation
-            className={`${classes.buttonAccept} notification-button-accept`}
-            disabled={inProgress}
-            onClick={onOpenInvitationDialog}
-          >
-            {t('button-accept')}
-          </Button>
-        )}
+        <Button
+          id={`notification-button-accept-${id}`}
```

Comment:
<!-- Suggestions propose improvements to the current subject. It's important to be explicit and clear on what is being suggested and why it is an improvement. Consider using patches to further communicate your intent.  -->
**🎯 suggestion (non-blocking):** ‏Could be removed unused id

Reply:
i need this for unitary test

Reply:
but i replace id by `id='notification-button-accept`